pkg/environment/environment.go

```diff
@@ -69,13 +71,49 @@ func RecoverEnvironmentErrors(errch chan<- error) {
 	errch <- nil
 }
 
-// GetInputChannels returns environment variable which contains the input channels
-func GetInputChannels() string {
+// GetInputChannels returns the input channels
+func GetInputChannels() []brokers.ChannelBroker {
+	return getChannelList(getRawInputChannels())
+}
+
+// GetOutputChannels returns the output channels
+func GetOutputChannels() []brokers.ChannelBroker {
```

Comment:
Similar thing here. "GetOutputChannels" seems that you're returning only a list of channels